mlir/lib/Conversion/ArithAndMathToAPFloat/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+add_mlir_library(ArithAndMathToAPFloatUtils
```

Comment:
Shouldn't this be?
```suggestion
add_mlir_library(MLIRArithAndMathToAPFloatUtils
```
This is the only MLIR library without the MLIR prefix.

Reply:
I can fix that in the next one

Reply:
It broke our downstream CI :)